crates/ruma-state-res/src/event_auth.rs

```diff
@@ -206,10 +206,11 @@ pub fn check_state_independent_auth_rules<E: Event>(
             ));
         };
 
-        // TODO:
-        //
         // Since v1, if there are entries which were themselves rejected under the checks performed
         // on receipt of a PDU, reject.
+        if auth_event.rejected() {
```

Comment:
Does this change need a changelog entry as well?

Reply:
It is comprised in the changelog entry that I added. It is a check on `auth_events`, I only did it in a separate commit to make smaller commits.